OptionSelector/DetailTableViewController.m

```diff
@@ -0,0 +1,67 @@
+//
+//  SubTableViewController.m
+//  OptionSelector
+//
+//  Created by Brian Blanco on 8/14/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "DetailTableViewController.h"
+
+@interface DetailTableViewController ()
+
+@end
+
+@implementation DetailTableViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    
+}
+
+- (void)didReceiveMemoryWarning {
+    [super didReceiveMemoryWarning];
+    // Dispose of any resources that can be recreated.
+}
+
+#pragma mark - Table view data source
+
+- (NSInteger)numberOfSectionsInTableView:(UITableView *)tableView {
+
+    return 1;
+}
+
+- (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
+    return self.category.options.count;
+}
+
+
+- (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    
+    UITableViewCell *cell = [tableView dequeueReusableCellWithIdentifier:@"reuseIdentifierDetail" forIndexPath:indexPath];
+   
+    
+    
+    NSString *animalOption = self.category.options[indexPath.row];
+    cell.textLabel.text = animalOption;
+    
+    if (self.category.options[indexPath.row] == self.category.selction) {
```

Comment:
You should always use `isEqualToString:` when comparing strings (even if they're statically defined).

Explanation: You can't rely on `==` when comparing Objective-C strings, because the same string could be represented by two distinct `NSString` instances.
```objc
NSString *string1 = [@[@"This is a string.", @"There are many like it,", @"but this one is mine."] componentsJoinedByString:@" "];
NSString *string2 = string1.copy;

// The two strings hold the same character sequence...
NSLog(@"String 1: %@", string1);
NSLog(@"String 2: %@", string2);

// ...but they are not the same instance of NSString
if (string1 == string2) {
	NSLog(@"equal");
} else {
	NSLog(@"not equal");
}
```